generated/ruby/google-cloud-ruby/google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1.rb

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I'm not sure I got your point. That does not mean this file has to be on this PR. Of course you will have to customize the local environment when you are trying to do that before everything is merged, but that does not justify the PR structures.